numba_cuda/numba/cuda/tests/nrt/test_nrt.py

```diff
@@ -77,6 +155,55 @@ def g(out_ary):
         self.assertEqual(out_ary[0], 1)
 
 
+class TestNrtLinking(CUDATestCase):
+    def run(self, result=None):
+        with override_config("CUDA_ENABLE_NRT", True):
+            super(TestNrtLinking, self).run(result)
+
+    def test_nrt_detect_linked_ptx_file(self):
+        src = f"#include <{get_include()}/nrt.cuh>"
+        src += """;
```

Comment:
The superfluous semicolon causes a warning.

```suggestion
        src += """
```